rust/lance/benches/scalar_index.rs

```diff
@@ -53,13 +53,21 @@ impl TrainingSource for BenchmarkDataSource {
 }
 
 impl BenchmarkFixture {
+    #[allow(dead_code)]
```

Comment:
Should we just remove it if it's dead?

Reply:
Let's keep it because we need to add tests for both v1 and v2 format after btree index supporting v2 format